### DISCUSSION TITLE
Discusion: What do you think is the biggest deal breaker in openrw project?

### DISCUSSION THREAD

**Discussion by ghost** [rwengine/openrw#767] [General]
Ofc there are many unimplemented things and many awful bugs. But maybe some are more important than others. 

Answers from users are highly valued and really welcomed. :)

**Comment by snshn** [comment]
GTA is all about cars and weapons, everything else is little different from a typical quest game, (GTA 3 was one of the first 3D open world games, but today it's not a surprise, hence I don't think that's worth mentioning, besides it's been implemented already). Here's what I think:

 - guns
   - properly working weapons (including things like scopes, grenade physics, weapon sounds, etc)
   - shootouts with (police) and mafia gangs
   - getting wasted from being shot
 - cars 
   - (police) chases
   - damaging and destroying vehicles
   - getting wasted from car explosions

Even after completing the game, people were still playing it "for fun", creating chaos in the city, hence I believe that that part of the gameplay, and not the storyline, is more important to implement first in order to keep the motivation to work on the project up and attract more contributions.

My opinion on what's of less importance:
 - for guns: carbomb, rocket launcher, sniper rifle
 - and for cars: boats, dodo, rhino, firetruck

I'd also like to mention that after being away from building and running the project for almost a year, I can see a tremendous amount of progress and it's really impressive what's already in the game. I'm not saying it's indistinguishable from the vanilla version, but I'm not worried about the future of the project since so much has been implemented, the completion of it seems to be inevitable, the only question I have is how many months/years later it'll be ready.
